app/assets/stylesheets/components/_button.scss

```diff
@@ -3,6 +3,14 @@
   font-size: $font-size-template;
 }
 
+@mixin button-white-text {
```

Comment:
Ensures buttons with white text remains white even if it's nested within `.TextContent`

Reply:
This doesn't feel quite right having a mixin include a class override. It seems like `.TextContent` is being a little too specific so perhaps that selector is worth revisiting.

Reply:
Yeah I agree, I don't like the specificity problem with `.TextContent`. We could possibly use `:not` selector. I had a try but was getting out-of-scope for this PR.

I'll raise a backlog ticket to revisit the best way to handle standard markdown v.s. non-standard elements `.TextContent` styling.